src/index/VocabularyGeneratorMain.cpp

```diff
@@ -0,0 +1,5 @@
+#include "./VocabularyGenerator.h"
+
```

Comment:
I thought this is integrated with the index building, or is this just for testing? That said there should be a `VocabularyGeneratorTest`